tiledb/cloud/udf.py

```diff
@@ -113,6 +116,9 @@ def exec_base(
     if timeout is not None:
         udf_model.timeout = timeout
 
+    if resource_class is not None:
+        udf_model.resource_class = resource_class
```

Comment:
Because the default here is `None` in both the function and the generated API code, I think you can put this just as

```python
udf_model = models.GenericUDF(
    # ...
    resource_class=resource_class,
)
```

instead of including it here.

Reply:
Correct. Yes the constructor does the same check.